test/ChainRulesTests.jl

```diff
@@ -0,0 +1,47 @@
+module ChainRulesTest
```

Comment:
Maybe consider adding some tests for (subset) of functions in `FUNCS_FROM_CHAINRULES`? We need to make sure rrules for these functions are correctly imported at least.

Reply:
I found all the functions in the `Base.FastMath` (the ones I listed) already work well in ReverseDiff (without importing the `rrule`s from ChainRules), so they are not necessary to be imported. Actually, almost all the numeric computing related functions are covered by ReverseDiff. Which ChainRules has but ReverseDiff doesn't are mostly numeric-unrelated ones, so I think we can just remove the file `src/chainrules.jl` and import nothing by default. How do you think about this? @yebai

Reply:
Sounds good, but it’s probably helpful to have a placeholder for functions imported from chain rules automatically. Maybe keep one function as a minimal in the list?